lib/money/bank/russian_central_bank.rb

```diff
@@ -58,7 +60,8 @@ def update_expired_at
       end
 
       def indirect_rate(from, to)
-        get_rate('RUB', to) / get_rate('RUB', from)
+        rate = original_get_rate('RUB', to).to_f / original_get_rate('RUB', from).to_f
+        rate.nan? || !!rate.infinite? ? nil : rate
```

Comment:
@olhor I think this will solve the issue - 

```
rate.nan? || !!rate.infinite? || rate == 0 ? nil : rate
```

Reply:
This is my proposal:
```ruby
def indirect_rate(from, to)
  dividend = original_get_rate('RUB', to)
  divider = original_get_rate('RUB', from)
  return nil if dividend.nil? || divider.nil?

  dividend.to_f / divider.to_f
end
```
Tests were green.

Reply:
@olhor Looks fine! 👍